doc/role_whitelist_management.md

```diff
@@ -246,6 +268,24 @@ without any record of that in the audit log.
 cassandra@cqlsh> ALTER ROLE ibbe WITH OPTIONS = { 'GRANT AUDIT WHITELIST FOR ALTER' : 'roles/jim' };
 ```
 
+### Grant Resources
+
+This resource type is used to represent an access derived whitelist on the wrapped resource.
+
+The operations that can be whitelisted depends on the type of resource being wrapped inside the grant.
+
+For example, when granting access derived whitelist __all__ on a __keyspace__ - _CREATE/ALTER/DROP/SELECT/MODIFY/AUTHORIZE_ operations will be whitelisted:
+```SQL
+cassandra@cqlsh> ALTER ROLE kalle WITH OPTIONS = { 'GRANT AUDIT WHITELIST FOR ALL' : 'grants/data/unit' };
+```
+
+And when granting access derived whitelist __all__ on __connections__ - _AUTHORIZE/EXECUTE_ operations will be whitelisted:
+```SQL
+cassandra@cqlsh> ALTER ROLE kalle WITH OPTIONS = { 'GRANT AUDIT WHITELIST FOR ALL' : 'grants/connections' };
+```
+
+A top-level grant resource (not containing any wrapped resource) can whitelist any operation (CREATE/ALTER/DROP/SELECT/MODIFY/AUTHORIZE/DESCRIBE/EXECUTE).
+Top-level grants can only be created by a user with the SUPERUSER flag.
 
 ## Permissions
 
```

Comment:
Another _really_ annoying thing with GitHub - I'm not able to put comments on _unchanged_ lines! WTF!?

Anyway, below in the Permissions section we should add "grants" to the list of things a super user can change.

Reply:
ok